pom.xml

```diff
@@ -51,6 +51,7 @@
         <dep.slice.version>0.27</dep.slice.version>
         <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
         <dep.tempto.version>1.18</dep.tempto.version>
+        <dep.zookeeper.version>3.4.6</dep.zookeeper.version>
```

Comment:
We are building with our own version of hadoop (through presto-hadoop-apache2) which is built with a zookeeper 3.4.8. In order to pass the dependency check for presto, we need to also override this property when we build. As far as I know, there is no way to override a version at build time unless it is extracted to a property.